cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -1,15 +1,17 @@
 import os
```

Comment:
This is not needed.

Reply:
It is needed for the path join according to the language.